src/test/java/de/tum/in/www1/artemis/util/UserTestService.java

```diff
@@ -803,7 +807,7 @@ public void testUserWithGroups() throws Exception {
         Course course = database.addEmptyCourse();
         courseRepository.save(course);
 
-        final var params = createParamsForPagingRequest("USER", "", "", Long.toString(course.getId()));
+        final var params = createParamsForPagingRequest("USER", "", "WITHOUT_REG_NO", "", Long.toString(course.getId()));
```

Comment:
I suggest using the Constant from UserRepository

Reply:
Adapted.